src/WLR/Util/Time.hsc

```diff
@@ -0,0 +1,13 @@
+module WLR.Util.Time where
```

Comment:
I think this shouldn’t be under `WLR.Util`; it should have its own top-level name, like `PIXMAN` does.

But… also, if I’m not mistaken, it looks like the `time` package [already defines bindings to `timespec`](https://github.com/haskell/time/blob/b621e9b85717618a27fce7e11bf70f6285f77931/lib/Data/Time/Clock/Internal/CTimespec.hsc#L22). Theirs seems to be non-exported, but for the future maybe we should investigate reusing that package somehow.

Reply:
I think I put it under WLR.Util because in the wlroots source tree it is at path `util/time.c`. I think I can move it to the toplevel

Reply:
But that’s a C source file; double-checking, it looks like the types are imported from the standard header file `<time.h>`.